packages/itwin/one-click-lca-widget/src/components/Reports.tsx

```diff
@@ -19,9 +19,14 @@ type CreateTypeFromInterface<Interface> = {
 
 type Reporting = CreateTypeFromInterface<Report>;
 
-const Reports = () => {
+export interface ReportProps {
```

Comment:
1. It's better to accept the whole client. This would allow users to provided their own client implementation / wrap it in some kind of caching, logging, or other additional logic. See how [Grouping & Mapping](https://github.com/iTwin/viewer-components-react/blob/master/packages/itwin/grouping-mapping-widget/src/WidgetShell/GroupingMappingWidget.tsx) widget does that.
2. The props interface needs to be exported from the package, not just from the module.